include/z64item.h

```diff
@@ -265,6 +284,9 @@ typedef enum {
     /* 0xFF */ ITEM_NONE = 0xFF
 } ItemID;
 
+#define BOTTLE_FIRST 0
+#define BOTTLE_MAX 6
```

Comment:
I'm kind of tempted to say we should have full fledged enum for each bottle along with a bottle max entry. Would help mirror the `SLOT_BOTTLE`s in the SLOT enum. Would like more opinions though.

Reply:
I think the problem with that is it's always iterated over, nothing in MM is put in a specific bottle.

Reply:
Yeah, we brainstormed that idea first of making an enum, but the inner enum values would go entirely unused afaik